app/models/computacenter_user_ledger.rb

```diff
@@ -0,0 +1,2 @@
+class ComputacenterUserLedger < ApplicationRecord
```

Comment:
Ledger generally refers to the whole book of record, not a single record, so the naming could potentially be misconstrued. Maybe something like `ComputacenterUserChange` or `ComputacenterUserUpdateEntry`. Naming is hard...

Reply:
Yea i thought about it for a while, I've now changed to `ComputacenterUserChange `